tests/unit/test_messages.py

```
@@ -1216,6 +1219,97 @@ def test_sync_messages_returns_message_response() -> None:
+        body += "data: [DONE]\n\n"
+        encoded = body.encode()
+
+        self.send_response(200)
+        self.send_header("Content-Type", "text/event-stream")
+        self.send_header("Content-Length", str(len(encoded)))
+        self.send_header("Connection", "close")
+        self.end_headers()
+        self.wfile.write(encoded)
+        self.wfile.flush()
+
+    def log_message(self, format: str, *args: Any) -> None:  # noqa: A002
```

Ruff (0.16.1)

[warning] Unused method argument: `format`

(ARG002)